mutiny-core/src/fees.rs

```diff
@@ -190,14 +189,6 @@ impl<S: MutinyStorage> FeeEstimator for MutinyFeeEstimator<S> {
 
         // any post processing we do after the we get the fee rate from the cache
         match confirmation_target {
-            ConfirmationTarget::MaxAllowedNonAnchorChannelRemoteFee => {
```

Comment:
https://github.com/lightningdevkit/rust-lightning/pull/2696

Reply:
Where do we do the "dust inflation limit" check now and does that cover it entirely?

Reply:
We have our dust limitation maxxed out https://github.com/MutinyWallet/mutiny-node/blob/481a8280fcf0d9248f2a80f286007c7dc3f27b3c/mutiny-core/src/node.rs#L2375

Shouldn't really matter for us because we don't route HTLCs so we aren't really vulnerable to an attack